flocker/control/_protocol.py

```diff
@@ -137,12 +163,32 @@ def connectionLost(self, reason):
         self.control_amp_service.disconnected(self)
 
 
+DEPLOYMENT_CONFIG = Field(u"configuration", repr, u"The cluster configuration")
+CLUSTER_STATE = Field(u"state", repr, u"The cluster state.")
```

Comment:
One description with a full stop, the other without.  These **aren't** sentences, so...